docs/Animations.md

```diff
@@ -49,11 +49,13 @@ class FadeInView extends React.Component {
     let { fadeAnim } = this.state;
 
     return (
-      <Animated.View                 // Special animatable View
-        style={{
-          ...this.props.style,
-          opacity: fadeAnim,         // Bind opacity to animated value
-        }}
+      <Animated.View // Special animatable View
+        style={[
+          {
+            opacity: fadeAnim // Bind opacity to animated value
+          },
+          ...this.props.style
```

Comment:
you can remove the spread if you are passing an array to style. 

this.props.style can be a number so spreading can still crash. passing array takes care of this.